app/v2/services/SubmitEndOfPeriodStatementService.scala

```diff
@@ -53,31 +55,31 @@ class SubmitEndOfPeriodStatementService @Inject()(connector: SubmitEndOfPeriodSt
 
   private val downstreamErrorMap: Map[String, MtdError] = {
     val errors = Map(
-      "INVALID_IDTYPE"                    -> InternalError,
+      "INVALID_IDTYPE"                    -> models.errors.InternalError,
```

Comment:
There seems to be no need to add "models.errors" before InternalError here. It can be removed from all places in this file.